airtable.js

```diff
@@ -80,44 +108,51 @@ const generateXdMarkup = (content) => {
     return [newsMarkDown, biosMarkdown];
 }
 
-fetchAirtablePromise(cacheFilePath, newsFilePath, biosFilePath)
-    .then((data) => {
+(async () => {
+    const newAirtableData = await fetchAirtablePromise(cacheFilePath, newsFilePath, biosFilePath);
+    const cacheData = JSON.parse(await fs.promises.readFile(cacheFilePath));
 
-        const cacheData = fs.readFileSync(cacheFilePath);
+    // Before we compare this data to cache, we need to sanitize the image paths
+    try {
+        await checkAndCleanImages(newAirtableData, cacheData);
+        console.log('Check images complete')
+    } catch (error) {
+        console.log('An error has occurred ', error);

```

Comment:
We could make some of these logging calls inside the `catch` statements in this file use the `error` method to better distinguish them from the regular log print outs:
```suggestion
        console.error('An error has occurred ', error);

```